modules/op-release-notes-1-4.adoc

```diff
@@ -0,0 +1,154 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-4_{context}"]
+= Release notes for {pipelines-title} General Availability 1.4
+
+
+[id="new-features-1-4_{context}"]
+== New features
+{pipelines-title} General Availability (GA) 1.4 is now available on {product-title} 4.7.
+
+[NOTE]
+====
+In addition to the stable/preview Operator channel for OCP 4.7, the general availability of Pipelines 1.4 comes with the deprecated channels corresponding to OCP 4.6 and OCP 4.5. However, support for the deprecated channels will be removed in the future releases.
+====
```

Comment:
Could we also emphasize that "and these deprecated channels will be removed in the following release of OpenShift Pipelines"?